sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Digital Twins Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Digital Twins Resources.")]
 
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
```

Comment:
You don't need to bump the `AsemblyVersion` if it's just a minor upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Reply:
Reverted this file